src/classes/VOL_CTRL_VolunteersJobListingFS.cls

```diff
@@ -56,6 +56,7 @@ global virtual with sharing class VOL_CTRL_VolunteersJobListingFS {
     global integer nMonthsToShow { get; set; }
     global integer nDaysToShow { get; set; }
     global date dtMonthFilter { get; set; }
+    global boolean eventAllShiftsInThePast { get; set; }
```

Comment:
please change this to jobAllShiftsInThePast, since V4S does not use the terminology Event.  thanks.